examples/ensemble/plot_adaboost_hastie_10_2.py

```diff
@@ -121,3 +155,11 @@
 leg.get_frame().set_alpha(0.7)
```

Comment:
In l.151, can you change `ax.set_xlabel("n_estimators")` by:

```python
ax.set_xlabel("Number of weak learners")
```

Reply:
Sounds good, will do